components/raftstore/src/store/peer.rs

```diff
@@ -1343,7 +1344,13 @@ impl Peer {
                     self.raft_group.skip_bcast_commit(true);
                     self.last_urgent_proposal_idx = u64::MAX;
                 }
-                let apply = Apply::new(self.region_id, self.term(), committed_entries);
+                let last_commit_index = self.get_store().committed_index();
```

Comment:
Could we pass the last index of `committed_entries ` to apply thread?

Reply:
Why? I don't see it's necessary.